radar/qa/operations.yaml

```diff
@@ -17,15 +17,13 @@ rationale: |
 license:
   commercial:
     company: Google
-    description: |
-      Use of this software requires a license for Google Cloud Platform.
```

Comment:
Why is the description deleted?

Reply:
This is the default description if you add "Google" as company. I'll probably try to clean this from all google entries.